examples/1.0.0/oauth.workflow.yaml

```diff
@@ -52,9 +52,12 @@ workflows:
         description: >-
           This is where you do the refresh
         operationId: get-token
-        parameters:
-          grant_type: refresh_token
-          refresh_token: $steps.do-the-auth-flow.outputs.my_refresh_token
+        requestBody:
+          payload:
```

Comment:
Actually (posting this AFTER my comment right below).. below you show a content-type: and payload: so I am thinking the payload above content-type is wrong?

Reply:
whoops - copy and past error